api/namespace/applications/routes.py

```diff
@@ -75,51 +82,50 @@ def post(self):
         account_id = args["account_id"]
```

Comment:
i think the marshal_with here should be using the model 'create_application' rather than 'application_full'